src/bottomSheets/bottomSheets.styles.scss

```diff
@@ -0,0 +1,46 @@
+@import '~styles/theme';
+
+$header-height: 2em;
+
+.bottomSheets {
+  bottom: 0;
+  left: 0;
+  margin: 0 2em;
+  position: fixed;
+  right: 0;
+}
+
+.sheets {
+  align-items: center;
+  background-color: $default-bg-color;
+  box-shadow: 0 0 5px 0 $font-color-dark;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  transform: translateY(calc(100% - #{$header-height}));
+  transition: transform .2s;
+
+  &.print {
+    transform: translateY(0);
+  }
+
+}
+
+.header {
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  height: $header-height;
+  width: 100%;
+}
+
+.content {
+  background-color: $default-bg-color;
+  justify-content: center;
+}
+
+.separator {
+  border: 0;
+  border-bottom: solid 1px $default-grey-color;
```

Comment:
em

Reply:
it's not relative, jsut one px it's good